resolwe_bio/processes/import_data/bam.yml

```diff
@@ -240,7 +190,7 @@
     resources:
       network: true
   data_name: '{{ src.file|default("?") }}'
-  version: 0.0.1
+  version: 0.1.0
   type: data:alignment:bam:secondary
```

Comment:
Why do we need this, why can't we just use a regular bam upload?

Reply:
Because we decided that all 3 data objects (original bam and 2 resulting bam-split objects) will live on a sample. We want to distinguish between them so they can be used in future analyses.